ioservice/io_foms.c

```diff
@@ -1861,6 +1861,7 @@ static int io_launch(struct m0_fom *fom)
 	} m0_tl_endfor;
 
 	m0_cob_put(container_of(file, struct m0_cob, co_file));
+	m0_cob_put(fom_obj->fcrw_cob);
```

Comment:
Please make sure the above two "cob" are not the same.
From the code, they are pointing to the same cob.

Reply:
Its pointing to the same cob.
Removed the duplicate call to m0_cob_put().

Reply:
OK